weblate/static/editor/full.js

```diff
@@ -324,6 +324,30 @@
         error: function (jqXHR, textStatus, errorThrown) {
           addAlert(errorThrown);
         },
+        success: function (data) {
+          if (dismiss_all) {
+            const { extra_flags, all_flags, ignore_check } = data;
+            $("#id_extra_flags").val(extra_flags);
+            $("#unit_all_flags").html(all_flags).css({
+              "background-color": "#ffa500ff",
+              display: "inline-block",
+              padding: "0px 6px",
+              "border-radius": "6px",
+            });
+            let transparency = 255;
+            let animation = setInterval(function () {
+              if (transparency === 0) {
+                clearInterval(animation);
+              } else {
+                let transpVal = transparency.toString(16).padStart(2, "0");
+                $("#unit_all_flags").css({
+                  "background-color": "#ffa500" + transpVal,
+                });
+                transparency--;
+              }
+            }, 15);
```

Comment:
Maybe it would be a better idea to use CSS transition instead of doing that manually in the JavaScript?

Reply:
done, created a class and added animation to CSS in [weblate/static/style-bootstrap.css](https://github.com/WeblateOrg/weblate/blob/main/weblate/static/style-bootstrap.css)